x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/cloud_summary_section.test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { coreMock } from '@kbn/core/public/mocks';
+import { render } from '@testing-library/react';
+import { expectIdsInDoc } from '../../../test/utils';
+import { DASHBOARD_COUNTER_CARDS } from '../test_subjects';
+import { CloudSummarySection } from './cloud_summary_section';
+import { mockDashboardData } from '../compliance_dashboard.test';
+import { TestProvider } from '../../../test/test_provider';
+import { screen } from '@testing-library/react';
+
+describe('<CloudSummarySection />', () => {
```

Comment:
I'd expect tests here to verify the rest of the functionality of the page (risks table, score chart, etc.) If you consider this out of scope please open a task

Reply:
yeah i'll add those tests here on every new component task, its already listed in the tasks